src/chapters/az-cli/walkthrough_az-cli.rst

```diff
@@ -0,0 +1,537 @@
+
+.. sourcecode:: powershell
+
+    > az group -h
+
+Take a moment to look over the output. Because resource groups are simple containers there isn't much beyond the standard CRUD commands.
+
+Planning
+--------
+
+Of the available commands we can see that our goal is to ``create`` the resource. Let's dig into the ``create`` Command to see what Arguments are available.
+
+.. sourcecode:: powershell
+
+    > az group create -h
+
+We will need to provide at minimum the two **required** arguments:
+
+- ``-n`` or ``-g``: the name of the resource group
+- ``-l``: the Azure Location
+
+Fortunately we have already configured our default Location value of ``eastus`` so all we need to supply is the resource group name. We will use a consistent naming convention to make sure none of our resources have conflicting names. The convention for resource groups we will use is "your name" + "cli-wt" (CLI walkthrough) + "rg" (resource group):
+
+    <name>-cli-wt-rg
+
+Note that there is a 23 character limit for resource names. If you have a long first name consider using a short name like Pat if your name is Patrick.
+
+Provisioning
+------------
+
+Now that we have determined the command structure and its arguments we can create our resource group:
+
+.. sourcecode:: powershell
+
+    > az group create -n <name>-cli-wt-rg
+
+You should see a JSON output like this:
+
+.. sourcecode:: bash
+
+    {
+        "id": "/subscriptions/<subscription ID>/resourceGroups/<name>-cli-wt-rg",
+        "location": "eastus",
+        "managedBy": null,
+        "name": "<name>-cli-wt-rg",
+        "properties": {
+            "provisioningState": "Succeeded"
+        },
+        "tags": null,
+        "type": "Microsoft.Resources/resourceGroups"
+    }
+
+Notice how the subscription and location are set automatically. The former by logging in and the latter by configuring its default value.
+
+Configuring
+-----------
+
+Just as we set a default location we will assign this resource group as a default as well. Be sure to enter your new resource group name as the value:
+
+.. sourcecode:: powershell
+
+    > az configure --default group=<name>-cli-wt-rg
+
+You can confirm the default has been set by checking the CLI configuration with the ``-l`` (list) argument and seeing that the "group" has value has been set correctly:
+
+.. sourcecode:: powershell
+
+    > az configure -l
+
+Virtual Machines
+================
+
+For this walkthrough we will not be using our VM to deploy an application but simply to get comfortable using the CLI. 
+
+Exploration
+-----------
+
+Virtual Machines are naturally more complex to interact with than a simple resource group. However, now that we understand the pattern of the ``az CLI`` that complexity can be managed using the help command to methodically work our way through its Sub-Groups and Commands.
+
+Once again let's begin by assessing what is available to us:
+
+.. sourcecode:: powershell
+
+    > az vm -h
+
+Planning
+--------
+
```

Comment:
You have Arguments capitalized twice in this section, however I don't think they should be.